rules/windows/persistence_evasion_registry_startup_shell_folder_modified.toml

```
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2021/03/15"
+maturity = "production"
+updated_date = "2021/03/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious startup shell folder modifications to change the default Startup directory in order to bypass
+detections monitoring file creation in the Windows Startup folder.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Startup Shell Folder Modification"
+note = "Verify file creation events in the new Windows Startup folder location."
+risk_score = 73
+rule_id = "c8b150f0-0164-475b-a75e-74b47800a9ff"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+registry where /* length(registry.data.strings) > 0 and */
```

```suggestion
registry where
```

The null check should be sufficient for this, yea?